lib/PuppeteerSharp.Tests/PageTests/Events/ErrorTests.cs

```diff
@@ -19,10 +19,10 @@ public async Task ShouldThrowWhenPageCrashes()
         {
             string error = null;
             Page.Error += (sender, e) => error = e.Error;
-            var gotoTask = Page.GoToAsync("chrome://crash");
+            await Page.GoToAsync("chrome://crash");
```

Comment:
The idea is not to await the task, so we wait for the error while the task is being executed.